.github/CODEOWNERS

```diff
@@ -309,6 +309,7 @@
 /core/templates/services/math-interactions.service.ts @iamprayush
 /extensions/domain*.py @brianrodri
 /extensions/interactions/ @iamprayush @vojtechjelinek
+/extensions/objects/templates/filepath-editor.component.* @kevintab95
 /extensions/objects/ @iamprayush
```

Comment:
You'll still be codeowning the above file i.e, the last pattern for the filename in the CODEOWNER file is used for assignment.

If it's an important file then you can move it to the CRITICAL path section below*

Reply:
Done.